AppCenterAuth/AppCenterAuthTests/MSAuthTests.m

```diff
@@ -1194,9 +1194,7 @@ - (void)testRefreshNeededWithNilAccountTriggersAnonymous {
   OCMStub([authMock retrieveAccountWithAccountId:fakeAccountId]).andReturn(nil);
   self.sut.authConfig = [MSAuthConfig new];
   self.sut.authConfig.authScope = @"fake";
-  OCMStub([authMock configAuthenticationClient]).andDo(^(NSInvocation *__unused invocation) {
-    self.sut.clientApplication = self.clientApplicationMock;
-  });
+  self.sut.clientApplication = self.clientApplicationMock;
```

Comment:
Could you please explain this change?

Reply:
configAuthenticationClient was not called, as loadConfigurationFromCache returned NO in this case. I've fixed it in the next commit.